_test/common_functions/FakeFAccess.m

```diff
@@ -0,0 +1,62 @@
+classdef FakeFAccess < sqw_binfile_common
+% A fake file access class that can return data as if from an sqw file
+
+properties
+    fake_data = [];
+    closed = false;
+end
+
+methods
+
+    function obj = FakeFAccess(data)
+        obj.fake_data = data;
+        obj.npixels_ = size(data, 2);
```

Comment:
this is not a property field of the class. Is this a legit alternative way to populate properties? And, while it is certainly pythonesque, do we want to do it here?

Reply:
Or is this a superclass field? Similarly for filename, filepath below.

Reply:
Yes, it's a superclass field. See `sqw_file_interface`.

Usually this value is read from file, so it needs to be set manually here.